src/Definition/Fixture/SimpleFixture.php

```diff
@@ -38,11 +38,11 @@
     private $specs;
 
     /**
-     * @var string
+     * @var string|int|FixtureInterface
      */
     private $valueForCurrent;
 
-    public function __construct(string $id, string $className, SpecificationBag $specs, string $valueForCurrent = null)
+    public function __construct(string $id, string $className, SpecificationBag $specs, $valueForCurrent = null)
```

Comment:
I would add the phpdoc for the param `$valueForCurrent` as well now then (just that param) besides in the var L41 like you did. The reason being an IDE will be able to check for the type during auto-completion